### PR TITLE
Many improvements/fixes

### PR DESCRIPTION
**Pull request by CPatchane**
* Minor webpack changes
* Fix css missing for mobile build
* Remove BarStore for redux only
* Upgrade cozy-ui for React compatibility
* Better apps fetching error handling
* Better public page handling
* Some linting

### CODE REVIEW

**Comment by CPatchane** [comment]
[6.6.1  Cozy-bar](https://trello.com/c/q9TYpwOj/458-661-cozy-bar)

File: src/components/Drawer.jsx [hunk 1]
```diff
@@ -32,17 +37,18 @@ class Drawer extends Component {
     this.props.drawerListener()
   }
 
-  async componentWillMount() {
-    await this.store.fetchSettingsData()
+  async componentDidMount() {
+    await this.props.fetchSettingsData()
```
Comment (gregorylegarec) [inline src/components/Drawer.jsx]: What about moving this call into `<Bar />` ?

File: src/components/Settings/index.jsx [hunk 2]
```diff
@@ -26,34 +30,36 @@ class Settings extends Component {
...
-    // display the loading spinner after BUSY_DELAY secs
-    const busySpinner = setTimeout(
-      () => this.setState({ busy: true }),
-      BUSY_DELAY
-    )
     // fetch data
-    await this.barStore.fetchSettingsData()
-    clearTimeout(busySpinner)
-    this.setState({ busy: false, opened: true })
+    await this.props.fetchSettingsData()
```
Comment (gregorylegarec) [inline src/components/Settings/index.jsx]: 😱

File: src/index.jsx [hunk 3]
```diff
@@ -15,9 +15,6 @@ import {
   setInfos
 } from './lib/reducers'
 
-// For now we have two stores, the goal is to transfer everythin
-// to the redux store
-import BarStore, { Provider as BarProvider } from './lib/BarStore'
```
Comment (gregorylegarec) [inline src/index.jsx]: :tada: 🏆

File: src/components/Apps/AppNavButtons.jsx [hunk 4]
```diff
@@ -3,11 +3,13 @@ import { connect } from 'react-redux'
 
 import { getHomeApp } from 'lib/reducers'
 
-import Icon from 'cozy-ui/react/Icon'
 import { translate } from 'cozy-ui/react/I18n'
 import homeIcon from 'assets/icons/icon-cozy-home.svg'
 import { isFetchingApps } from 'lib/reducers'
 
+import IconTop from 'assets/icons/24/icon-top.svg'
+import IconBottom from 'assets/icons/24/icon-bottom.svg'
```
Comment (gregorylegarec) [inline src/components/Apps/AppNavButtons.jsx]: Note for later : is it possible to get directly those icons from cozy-ui ?

**Comment by gregorylegarec** [comment]
Great, LGTM except a couple of thing we discussed during pair reviewing 🍻

**Comment by CPatchane** [comment]
The commit 
https://github.com/cozy/cozy-bar/pull/332/commits/d58bd1e5432b738bc5393936f8c354ee0fab7d0e is related to https://github.com/cozy/cozy-ui/issues/674

**Pull request merged**